### DISCUSSION TITLE
Pure RAM madmax

### DISCUSSION THREAD

**Discussion by prodchia** [madMAx43v3r/chia-plotter#885] [Q&A]
Has anyone tried running madmax with temp1 and temp2 both as RAM disk? Is this possible with 256GB RAM?

**Comment by idog8818** [comment]
Yes. I'm running on 420G radisk. 480 in total.  Don't separate temp and 2.  All seems great. Each plot takes 40 mins. If plotting 2 same time. 60mins..

I think it's because of I'm running Windows and ram at 1800mhz.

**Comment by idog8818** [comment]
Updates: Running in UBuntu, same device, 22mins per plot.

**Reply by svennand** [reply]
Have you tested with temp drive beeing ssd. I was thinking in comparison to using just ram vs ram+ssd.
What cpu are you using?

**Reply by bradkrane** [reply]
I get 22-23min plots using M2 for tmp1 and 105G RAM disk for temp2 bare Linux. Going to try Windows eventually but haven't yet

**Comment by prodchia** [comment]
Thanks for the update. I only have 256GB, and my question was if it was possible to use MADMAX with temp1 and temp2 as 240GB RAM disk.

**Reply by idog8818** [reply]
Unlikely. it will require min 300G for both. But you still can use 2 for 110G. Since I have 480G I've tried using 300G for a longtime. Until I realize if the copy cannot complete on time.which it will occupy 101G. the app will exit. that's why I'm using 420G. 20G is the buffer

**Reply by rusliko** [reply]
> Unlikely. it will require min 300G for both. But you still can use 2 for 110G. Since I have 480G I've tried using 300G for a longtime. Until I realize if the copy cannot complete on time.which it will occupy 101G. the app will exit. that's why I'm using 420G. 20G is the buffer

Specify, please, do you use one temporary directory? That is, I have 384 GB of frames. I am using two frame discs, 220 and 110. Can I use one? How? Thanks in advance for the hint!

**Reply by idog8818** [reply]
yes you can,  say ramdisk is R drive and use  -t R:\ -2 R:\

**Reply by idog8818** [reply]
make your ramdisk say 330g to give it 10% buffer for space if you want.
